library/templates/dcp_mappluto_clipped.yml

```diff
@@ -0,0 +1,55 @@
+dataset:
+  name: &name dcp_mappluto_clipped
+  version: "{{ version }}"
+  acl: public-read
+  source:
+    url:
+      path: https://www1.nyc.gov/assets/planning/download/zip/data-maps/open-data/nyc_mappluto_{{ version }}_shp.zip
```

Comment:
this link is invalid
<img width="1460" alt="Screen Shot 2022-09-07 at 10 47 23 AM" src="https://user-images.githubusercontent.com/41444592/188908325-98d12145-b42f-4f1f-8a8e-87f934c916de.png">

Reply:
hold on, I think I know what I did wrong here